app/poros/search_result.rb

```diff
@@ -0,0 +1,42 @@
+class SearchResult
+
+    def repos(token)
+      conn = Faraday.new('https://api.github.com') do |faraday|
```

Comment:
This conn bit is where we'll want to refactor into a GitHub service object, but this is a great refactor to start with for all three of these methods.